pkg/metric_generator/generator.go

```diff
@@ -52,6 +56,11 @@ func NewFamilyGenerator(name string, help string, metricType metric.Type, deprec
 	return f
 }
 
+// NewFamilyGenerator creates new FamilyGenerator instances.
+func NewFamilyGenerator(name string, help string, metricType metric.Type, deprecatedVersion string, generateFunc func(obj interface{}) *metric.Family) *FamilyGenerator {
+	return NewFamilyGeneratorWithStability(name, help, metricType, basemetrics.ALPHA, deprecatedVersion, generateFunc)
```

Comment:
I'm not sure why all metrics are being set to `alpha`?

Reply:
Not all. 
In replacing NewFamilyGenerator with NewFamilyGeneratorWithStability. 

For example: https://github.com/kubernetes/kube-state-metrics/pull/1844/files#diff-28c04d8ad4d0fd23a2b14d61e94ca54e757c1e5ab971cea37974fe1b2f260085

Reply:
Ah, okay. So for now we are setting all `node` metrics to `alpha`?

Reply:
It seems only two of these metrics were experimental before. So IIUC, if the `node` metrics are being set to `alpha`, should it only be for these two?
```
| kube_node_annotations        | Gauge       | Kubernetes annotations converted to Prometheus labels
| kube_node_role               | Gauge       | The role of a cluster node
```

Reply:
> what state were they in before? Experimental?

Yes, we are mapping into k8s stability level:
Experimental -> Alpha
Stable -> Stable

> So IIUC, if the node metrics are being set to alpha, should it only be for these two?

Working in progress for other Stable metrics. Because there are many test codes need to be changed. 
I planned to create 2+ CLs. The first CL (this CL) to import stability framework, other CLs migrate NewFamilyGenerator 
 to NewFamilyGeneratorWithStability.

Reply:
Added all STABLE metrics just now.

Reply:
Oh wow! This looks amazing, thank you! 🎉

Reply:
Oh, only stable metrics are using NewFamilyGeneratorWithStability now.

Will create another CL to migrate non-stable metrics to use NewFamilyGeneratorWithStability. Because this way has less merge conflicts in this CL.